package.json

```diff
@@ -47,8 +47,8 @@
     "devDependencies": {
         "concurrently": "^4.1.0",
```

Comment:
Have you considered also setting specific versions of Concurrently and Diff to prevent issues coming from different versions of the libraries installed on different environments?

Reply:
I don't know about Concurrently but maybe for Diff you're right. The problem in this case was that `mocha` library was using a version of diff which was containing a vulnerability.

Reply:
That sounds like a good idea to prevent some bugs that are hard to reproduce due to differing versions on different environments

Reply:
@balhar-jakub Hi Jakub, sorry I think I've accidentally remove your commit because I was rebasing master and I forgot to update the branch. Anyway, I think you had only applied the changed that I've done in the last commit. If there was something else, please let me know, otherwise it's ready to be merged. Thank you!